polaris.shopify.com/src/utils/markdown.mjs

```diff
@@ -66,6 +68,40 @@ export const parseMarkdown = (inputMarkdown) => {
     });
   }
 
+  // Add some custom HTML to <!-- colors --> tags
+  const colorsRegex = /<!-- (colors) -->/gis;
+  if (markdown.match(colorsRegex)) {
+    markdown = markdown.replace(colorsRegex, (match) => {
+      const colorOrder = [
+        'gray',
+        'green',
+        'teal',
+        'blue',
+        'purple',
+        'red',
+        'orange',
+        'yellow',
+      ];
+
+      return colorOrder.reduce((acc, color) => {
+        const shades = colors[color] ?? [];
+        const swatches = Object.entries(shades)
+          .sort(([prevShade], [nextShade]) =>
+            Number(prevShade) < Number(nextShade) ? 1 : -1,
+          )
+          .map(
+            ([shade, value]) =>
+              `<div><div class="colors-swatch" style="background-color: ${value};"></div><div>${shade}</div></div>`,
```

Comment:
```suggestion
              `<div class="colors-swatch" style="background-color: ${value};"></div><div>${shade}</div>`,
```

Reply:
I think this is needed in order to group the color swatch with the shade text. Otherwise, the shade text would become the next column following the swatch. I had a class name to help identify this div's purpose (`color-column`) but removed it since it didn't have any styles.